SPEC.md

```
@@ -0,0 +1,1065 @@
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
+```
+
+### 10.2 Hosting (Vercel)
+
+#### Project Configuration
+
+| Field          | Value                                  |
+| -------------- | -------------------------------------- |
+| Dashboard      | https://vercel.com/laststance/gitbox   |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)